sdks/python/test-suites/portable/common.gradle

```diff
@@ -206,7 +206,10 @@ def createPrismRunnerTestTask(String workerType) {
   def taskName = "prismCompatibilityMatrix${workerType}"
 
   def prismBin = "${rootDir}/runners/prism/build/tmp/prism"
-  def options = "--prism_bin=${prismBin} --environment_type=${workerType}"
+  def options = "--prism_bin=${prismBin}"
+  if (workerType != 'LOOPBACK') {
+    options += " --environment_type=${workerType}"
```

Comment:
Is this change because something weird was going on when environment type was "docker"?

Reply:
I think this was an artifact of trying to run as a separate gRPC service.